--- a/nbdime/diffing/notebooks.py
+++ b/nbdime/diffing/notebooks.py
@@ -553,7 +553,7 @@
                     del notebook_differs[key]
             else:
                 notebook_differs[key] = diff_ignore_keys(
-                    inner_differ=diff, ignore_keys=['collapsed', 'autoscroll', 'deletable', 'editable'])
+                    inner_differ=diff, ignore_keys=['toc', 'collapsed', 'autoscroll', 'deletable', 'editable'])
     else:
         for key in metadata_keys:
             notebook_differs[key] = diff_ignore
